spec/spec_helper_system.rb

```diff
@@ -19,7 +19,7 @@
 
     # Install modules and dependencies
     puppet_module_install(source: proj_root, module_name: 'splunk')
-    shell('puppet module install nanliu-staging --version 0.3.1')
+    shell('puppet module install puppet-archive --version 2.0.0')
```

Comment:
btw, it seems like this is for https://github.com/puppetlabs/rspec-system, which is deprecated and presumably not in use now that the module has beaker based acceptance tests

Reply:
No Longer applicable